docs/background-image.md

```diff
@@ -4,10 +4,6 @@
 
 Utilities for controlling an element's background image.
 
-::: warning Unsupported
-This functionality is not yet supported! If you need this, reach out to us on [#nmp-warp-design-system](https://sch-chat.slack.com/archives/C04P0GYTHPV).
-:::
-
 ## Quick reference
 
 <qr-table />
```

Comment:
Do we want to keep the `! TODO: Add support for bg-none` in line 11, too?

Reply:
Nope. Especially not after adding it here: https://github.com/warp-ds/drive/pull/192 😄